### PR TITLE
Add build option

### PR DESCRIPTION
**Pull request by wewang58**
@openshift/devexp-qe  PTAL thanks

### CODE REVIEW

File: lib/rules/cli/4.5.yaml [hunk 1]
```diff
@@ -1091,6 +1091,7 @@
   :options:
     :buildconfig: <value>
     :build_arg: --build-arg=<value>
+    :build_loglevel: --build-loglevel=<value>
```
Comment (xiuwang) [inline lib/rules/cli/4.5.yaml:1094]: How about 4.4.yaml file, do we need update this too?

Comment (wewang58) [reply lib/rules/cli/4.5.yaml:1094]: Yes, missed it, Updated 4.4 yaml now.

**Pull request merged**